msal-client-credentials-flow/README.md

```diff
@@ -0,0 +1,139 @@
+-->
+
+This sample Node.js application demonstrates how to issue a call to a protected API using the client credentials flow.  A request will be issued to Microsoft Graph using the application's own identity.
+```console
+$ curl http://localhost:8080/users
+{
+  "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#users/$entity",
+  "businessPhones": ["+1 (999) 5551001"],
+  "displayName": "Contoso Employee",
+  "givenName": "Contoso",
+  "jobTitle": "Worker",
+  "mail": "cemployee@contoso.com",
+  "mobilePhone": "1 999-555-1001",
+  "officeLocation": "Contoso Plaza/F30",
+  "preferredLanguage": null,
+  "surname": "Employee",
+  "userPrincipalName": "contoso_employee@contoso.com",
+  "id": "e3a49d8b-d849-48eb-9947-37c1f9589812"
+}
+
+```
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Node.js 16+
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Configure an application to expose a web API](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-expose-web-apis) to register the sample API.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting    | Value for this sample app                                                    | Notes                                                                                              |
+|:---------------------------------:|:-----------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                          | `Node Web API`                                                               | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**       | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**                 | _None_                                                                       | No redirect URI required; don't select a platform.                                                                    |
+| **Client secret**                 | _**Value** of the client secret (not its ID)_                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+| **API Permissions**               | `User.Read.All`                                                              | Add an Application Permission for Microsoft Graph. Grant admin consent. Required value for this sample.      |
```

Comment:
Yeah, probably don't want to do that.  Getting that permission can be hard in some orgs.  We avoided this in other client credential examples for this reason.

Reply:
Removed User.Read.All